Riot/Modules/Common/Models/Section.swift

```diff
@@ -41,12 +41,18 @@ final class Section: NSObject {
         addRow(Row.row(withTag: tag))
     }
     
+    func addRows(withCount count: Int) {
+        for i in 0..<count {
+            addRow(Row.row(withTag: i))
+        }
+    }
```

Comment:
Yes the interest of `Section` is to allow to keep usage of an enum value as a row identifier. But in the case of a list of objects there is no interest to use specific tag here. In the case of list of sessions you just want a number of rows. 
Otherwise if we want to have a similar approach like we are doing in Swift, we should force to use "tag" associated to a kind of cell and do not rely on index but only on tags. This is better but we should rewrite our code, for example in `cellForRow`:
```
else if (tableSection.tag == SECTION_CRYPTO_SESSIONS)
        {
                switch (tableRow.tag) {
                    case kCryptoSessionDeviceInfoRow:
                        cell = [self deviceCellWithDevice:devicesArray[row] forTableView:tableView];
                        break;
                    case kCryptoSessionLoadingInfoPart1Row:
                        cell = [self descriptionCellForTableView:tableView
                        withText:NSLocalizedStringFromTable(@"security_settings_crypto_sessions_loading", @"Vector", nil) ];
                        break;
                    case kCryptoSessionLoadingInfoPart2Row:
                        cell = [self descriptionCellForTableView:tableView
                        withText:NSLocalizedStringFromTable(@"security_settings_crypto_sessions_description_2", @"Vector", nil) ];
                        break;
                    default:
                        cell = nil;
                }
            }
        }
```